### ISSUE TITLE
Not Show "ingress-nginx-controller" pod after enable ingress addon in minikube

### ISSUE THREAD

**Issue by Shubham82** [kubernetes/website#28475]
**This is a Bug Report**

**Problem:**
In [Set up Ingress on Minikube with the NGINX Ingress Controller](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/) When performing Enable the Ingress controller by running the command  `minikube addons enable ingress` then it doesn't show `ingress-nginx-controller-xxxx` pod in kube-system namespace, as shown below
```
kubectl get pod -n kube-system
NAME                                READY   STATUS    RESTARTS   AGE
coredns-558bd4d5db-qbljq            1/1     Running   0          3m31s
etcd-minikube5                      1/1     Running   0          4m14s
kube-apiserver-minikube5            1/1     Running   0          4m14s
kube-controller-manager-minikube5   1/1     Running   1          4m14s
kube-proxy-lvjl7                    1/1     Running   0          3m32s
kube-scheduler-minikube5            1/1     Running   0          4m14s
storage-provisioner                 1/1     Running   1          3m32s
```

 Instead of that it create `ingress-nginx-controller-xxxx`  pod in ingress-nginx namespace (namespace created when we enable ingress addon) as  shown below:
```
kubectl get pod -n ingress-nginx
NAME                                        READY   STATUS      RESTARTS   AGE
ingress-nginx-admission-create-dzlrn        0/1     Completed   0          92s
ingress-nginx-admission-patch-rwbbl         0/1     Completed   1          92s
ingress-nginx-controller-59b45fb494-hdh56   1/1     Running     0          93s
```
 
**Proposed Solution:**
It should show the output of **kubectl get pod -n ingress-nginx** instead of **kubectl get pod -n kube-system**
**Page to Update:**
https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/

Kubernetes Version:
v1.21.0
Minikube Version:
v1.21.0
Reference [PR](https://github.com/kubernetes/minikube/pull/10879)

**Comment by jihoon-seo** [comment]
Hello @Shubham82

The PR #28154 and #28355 are also suggesting this change.

/hold

**Comment by Shubham82** [comment]
Thanks, @jihoon-seo for the information.

**Comment by Shubham82** [comment]
The issue is fixed in PR #28355
/close
